sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -12,10 +13,6 @@ node-ip.setting-generator = "pluto node-ip"
 provider-id.setting-generator = "pluto provider-id"
 affected-services = ["kubernetes"]
 
-[metadata.settings.kubernetes.pod-infra-container-image]
-setting-generator = "schnauzer-v2 render --requires 'aws@v1' --requires 'kubernetes@v1(helpers=[pause-prefix])' --template '{{ pause-prefix settings.aws.region }}/eks/pause:3.1-eksbuild.1'"
-affected-services = ["kubernetes", "containerd"]
-
```

Comment:
e3bc71dad7734070f62c7541626edc12c76e5c0d is an example of where we've deprecated a setting and turned it into a no-op. Metadata here could be:
```
[metadata.settings.kubernetes.pod-infra-container-image]
affected-services = ["pod-infra-container-image"]

[services.pod-infra-container-image]
configuration-files = ["pod-infra-container-image-log-message"]
restart-commands = ["systemctl restart deprecation-warning@pod-infra-container-image.timer"]

[configuration-files.pod-infra-container-image-log-message]
path = "/etc/deprecated-settings/pod-infra-container-image"
template-path = "/usr/share/templates/pod-infra-container-image"
```

Then you should just need to add a `packages/kubernetes-*/pod-infra-container-image` template:
```
[required-extensions]
kubernetes = { version = "v1", optional = true }
+++
{{~#if settings.kubernetes.pod-infra-container-image~}}
DEPRECATED_SETTING=settings.kubernetes.pod-infra-container-image
{{~/if~}}
```

And in the spec:
```
SourceXYZ: pod-infra-container-image

%install
...
install -p -m 0644 %{S:XYZ} %{buildroot}%{_cross_templatedir}/pod-infra-container-image

%files -n %{_cross_os}kubelet-1.XY
...
%{_cross_templatedir}/pod-infra-container-image
```

This will leverage the common `deprecated-warning@service` and `deprecated-warning@timer` implementation in `release`.